src/Microsoft.Identity.Client/Internal/Broker/BrokerInteractiveRequest.cs

```diff
@@ -38,9 +38,9 @@ internal class BrokerInteractiveRequest
 
         public async Task<MsalTokenResponse> SendTokenRequestToBrokerAsync()
         {
-            Broker = brokerFactory.CreateBrokerFacade(_serviceBundle);
+            _broker = _brokerFactory.CreateBrokerFacade(_serviceBundle);
```

Comment:
This is not a facade, see facade design pattern here https://stackoverflow.com/questions/5242429/what-is-the-facade-design-pattern

A factory creates objects, the most natural method on a factory is "Create"